KsApi/fragments/CategoryFragment.graphql

```diff
@@ -5,5 +5,6 @@ fragment CategoryFragment on Category {
   parentCategory {
     id
     name
+    analyticsName @include(if: $withParentCategoryAnalyticsName)
```

Comment:
in what situation are we calling false here?

Reply:
Yea so I noticed that the `parentCategory` `analyticsName` added about 0.5 seconds of lag to our fetch project call above, so I didn't want to add it blindly to a pre-existing fragment used by many queries (fetch project, fetch backing, fetch addons), so I included it optionally. The only queries that set it to true should be the new ones (`fetchCategories` and `fetchCategory`) every other call should have it set to `false`